packages/sources/amberdata/src/adapter.ts

```diff
@@ -48,6 +48,7 @@ interface Message {
   }
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
 export const makeWSHandler = (defaultConfig?: DefaultConfig): MakeWSHandler<Message | any> =>
```

Comment:
Would it be better to do Message | UnknownWSMessage instead, so we avoid disabling the any check?